daemon/archive.go

```diff
@@ -375,7 +375,7 @@ func (daemon *Daemon) CopyOnBuild(cID, destPath, srcRoot, srcPath string, decomp
 
 	destExists := true
 	destDir := false
-	rootUID, rootGID := daemon.GetRemappedUIDGID()
+	rootIDs, _ := daemon.idMappings.RootPair()
```

Comment:
Looking at the setup code again I notice this feature is not support on windows, so we don't have to worry about that case.

Remapping to the root user is already checked in `NewIDMappings()`, so I'm going to remove the error on `RootPair()`.